api/api_private_account.go

```diff
@@ -233,22 +233,17 @@ func (s *PrivateAccountAPI) signTransaction(ctx context.Context, args SendTxArgs
 // tries to sign it with the key associated with args.From. If the given passwd isn't
 // able to decrypt the key it fails.
 func (s *PrivateAccountAPI) SendTransaction(ctx context.Context, args SendTxArgs, passwd string) (common.Hash, error) {
-	if args.TypeInt != nil {
-		if args.TypeInt.IsFeeDelegatedTransaction() {
-			return common.Hash{}, errNotForFeeDelegationTx
-		}
-	}
```

Comment:
The argument check will be implemented later soon. 
Currently, this check is not necessary. It only gives detailed error messages to users.